### PR TITLE
Enhancing Winlogbeat docs

### PR DESCRIPTION
**Pull request by andrewkroh**
- Changed RemoteSigned to UnRestricted because I found a case where RemoteSigned didn't work
- Added details on how to use list event log channels and configure them
- Added a note to the SID field explaining why user.name, user.domain, user.type may not be present

### CODE REVIEW

File: winlogbeat/docs/getting-started.asciidoc [hunk 1]
```diff
@@ -70,11 +72,15 @@ logging:
   level: info
 --------------------------------------------------------------------------------
 
+For instructions on how to obtain a list of available event logs see the
+configuration details for
+<<configuration-winlogbeat-options-event_logs-name,event_logs.name>>.
```
Comment (andrewkroh) [inline winlogbeat/docs/getting-started.asciidoc]: Is this the proper way to link this?

Comment (dedemorton) [inline winlogbeat/docs/getting-started.asciidoc]: Looks right to me. You really only need to specify the first part, though, unless you want the link text to differ from the heading text. So you could have just:
<<configuration-winlogbeat-options-event_logs-name>> 

But I'd just leave it as it is.

BTW the doc looks good. If you ever decide you want to be a technical writer, hit us up. :-)

**Comment by monicasarbu** [comment]
LGTM

File: winlogbeat/docs/getting-started.asciidoc (hunk 1 above)
Comment (andrewkroh) [inline winlogbeat/docs/getting-started.asciidoc]: Thanks @dadoonet

**Pull request merged**